build/openwisp_freeradius/raddb/sites-enabled/default

```diff
@@ -0,0 +1,59 @@
+server default {
+    api_token_header = "Authorization: Bearer $ENV{ORG_UUID} $ENV{FREERADIUS_TOKEN}"
+
+    listen {
+        type = auth
+        ipaddr = *
+        port = 0
+        limit {
+            max_connections = 16
+            lifetime = 0
+            idle_timeout = 30
+        }
+    }
+
+    listen {
+        ipaddr = *
+        port = 0
+        type = acct
+        limit {}
+    }
+
+    authorize {
+        update control { &REST-HTTP-Header += "${...api_token_header}" }
+        rest
+        sql
+        dailycounter
+        noresetcounter
+        dailybandwidthcounter
+    }
+
+    authenticate {}
+
+    preacct {
+        preprocess
+        acct_unique
+        suffix
+        files
+    }
+
+    accounting {
+        update control { &REST-HTTP-Header += "${...api_token_header}" }
+        rest
+    }
+
+    session {}
+
+    post-auth {
+        update control { &REST-HTTP-Header += "${...api_token_header}" }
+        rest
+
+        Post-Auth-Type REJECT {
+            update control { &REST-HTTP-Header += "${....api_token_header}" }
+            rest
+        }
+    }
+
+    pre-proxy {}
+    post-proxy {}
+}
```

Comment:
this one for example, not only may vary from user to user, but each organization may need more than one site.
the thing is that this is more configuration than code, I'm not sure if it's right to include this in the image, from a certain point of view we do need to include something in the image if we want it to work out of the box, but I worry that if we don't provide a way to override these freeradius configurations, the project will not be as useful as we want it to be

Reply:
The default `rabbd/` files are just sample of basic configuration for testing (and can be used as an example for user), user can mount their own raddb directory for customization. :smile: